src/azure-cli-core/azure/cli/core/extension/__init__.py

```diff
@@ -307,7 +307,7 @@ def get_extension_path(ext_name):
 
 
 def get_extensions(ext_type=None):
-    logger.debug("Extensions directory: '%s'", EXTENSIONS_DIR)
+    # logger.debug("Extensions directory: '%s'", EXTENSIONS_DIR)
```

Comment:
You can delete this line to make code neat.

Reply:
removed